strongbox-data-service/src/main/java/org/carlspring/strongbox/gremlin/repositories/GremlinVertexRepository.java

```diff
@@ -0,0 +1,59 @@
+package org.carlspring.strongbox.gremlin.repositories;
+
+import java.util.function.Supplier;
+
+import javax.transaction.Transactional;
+
+import org.apache.tinkerpop.gremlin.structure.Vertex;
+import org.carlspring.strongbox.data.domain.DomainObject;
+import org.carlspring.strongbox.gremlin.adapters.UnfoldEntityTraversal;
+import org.carlspring.strongbox.gremlin.dsl.EntityTraversal;
+import org.carlspring.strongbox.gremlin.dsl.EntityTraversalSource;
+
+@Transactional
+public abstract class GremlinVertexRepository<E extends DomainObject> extends GremlinRepository<Vertex, E>
```

Comment:
Could you please, add some documentation here as well? Thanks!

Reply:
done